fe/src/app/dashboard/page.tsx

```diff
@@ -1,28 +1,28 @@
 import React from "react";
-import { getSession, getAccessToken } from "@auth0/nextjs-auth0";
```

Comment:
This could be left instead of two separate import statements of the same library.